python/dgl/graphbolt/minibatch.py

```diff
@@ -6,11 +6,97 @@
 import torch
 
 import dgl
+from dgl.heterograph import DGLBlock
 
 from .base import etype_str_to_tuple
 from .sampled_subgraph import SampledSubgraph
 
-__all__ = ["MiniBatch"]
+__all__ = ["DGLMiniBatch", "MiniBatch"]
+
+
+@dataclass
+class DGLMiniBatch:
```

Comment:
Move this blow Minibatch.

Reply:
MIniBatch need to use it so it has to be defined ahead.